src/queries/makeQueries.ts

```diff
@@ -83,6 +85,26 @@ Example:
   return waitForOptions;
 }
 
+function getWrongHostComponentNamesError(): string {
+  const reactNativeHostComponentNames = getReactNativeHostComponentNames();
+  const configHostComponentNames = getConfig().hostComponentNames;
```

Comment:
```suggestion
  const hostComponentNamesConfig = getConfig().hostComponentNames;
```

Reply:
Not sure I like this change, I think of this rather as the names in the config over the config for names so `configHostComponentNames ` is better in that way to me, not sure if you see what I mean

Reply:
yes i understand, i misread it